src/pvr2wmc.cpp

```diff
@@ -718,16 +717,17 @@ PVR_ERROR Pvr2Wmc::GetChannelGroupMembers(ADDON_HANDLE handle, const PVR_CHANNEL
 	if (IsServerDown())
 		return PVR_ERROR_SERVER_ERROR;
 
-	CStdString request;
-	request.Format("GetChannelGroupMembers|%s|%s", group.bIsRadio ? "True" : "False", group.strGroupName);
-	vector<CStdString> results = _socketClient.GetVector(request, true);
+	std::string request;
+	//request.Format("GetChannelGroupMembers|%s|%s", group.bIsRadio ? "True" : "False", group.strGroupName);
```

Comment:
Why do you want to keep all these `foo.Format` comments? Cannot see any benefit. The line that follows this comment is always basically the same, so the comment is of no additional value. Applies to many places in this source file...

Reply:
I left them because I have security issues.  The .Format call was from cstdstring and I worried about the changeover.  I'll get over it (and get rid of them)